src/cnd_settings.rs

```diff
@@ -23,11 +22,42 @@ pub struct Network {
     pub listen: Vec<String>,
 }
 #[derive(Clone, Debug, Serialize)]
-pub struct HttpSocket {
+pub struct HttpApi {
+    pub socket: Socket,
+    pub cors: Cors,
+}
+
+#[derive(Clone, Debug, Serialize)]
+pub struct Socket {
     pub address: IpAddr,
     pub port: u16,
 }
 
+#[derive(Clone, Debug, Serialize)]
+pub struct Cors {
+    pub allowed_origins: AllowedOrigins,
+}
+
+#[derive(Clone, Debug, Serialize)]
+#[serde(untagged)]
+pub enum AllowedOrigins {
+    All(All),
+    None(None),
+    Some(Vec<String>),
+}
+
+#[derive(Clone, Debug, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum All {
+    All,
+}
+
+#[derive(Clone, Debug, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum None {
```

Comment:
Create-comit-app is only ever using the `All` variant. Do we really need to copy over all the code from `comit-rs`?

Reply:
Is there not a simpler way of achieving what we need for this usecase?